Dockerfile

```diff
@@ -53,28 +60,34 @@ COPY . .
 # To restore workspaces symlinks
 RUN yarn install --frozen-lockfile --verbose
 
-ENV PATH /usr/src/app/node_modules/.bin:$PATH
-ENV QUICK_BUILD true
-ENV PUBLIC_URL /ohif-viewer/
+ENV PATH=/usr/src/app/node_modules/.bin:$PATH
+ENV QUICK_BUILD=true
+ENV PUBLIC_URL=/ohif-viewer/
 # ENV GENERATE_SOURCEMAP=false
 # ENV REACT_APP_CONFIG=config/default.js
 
 RUN yarn run build
 
 # Stage 3: Bundle the built application into a Docker container
-# which runs Nginx using Alpine Linux
-FROM nginxinc/nginx-unprivileged:1.25-alpine as final
+# which runs Nginx built on top of Amazon Linux
+
+FROM quay.io/cdis/python-nginx-al:master AS final
 #RUN apk add --no-cache bash
 ENV PORT=8080
-RUN rm /etc/nginx/conf.d/default.conf
-USER nginx
-COPY --chown=nginx:nginx .docker/Viewer-v3.x /usr/src
+USER gen3
+COPY --chown=gen3:gen3 .docker/Viewer-v3.x /usr/src
 RUN chmod 777 /usr/src/entrypoint.sh
 COPY --from=builder /usr/src/app/platform/app/dist /usr/share/nginx/html
 # In entrypoint.sh, app-config.js might be overwritten, so chmod it to be writeable.
 # The nginx user cannot chmod it, so change to root.
 USER root
-RUN chmod 666 /usr/share/nginx/html/app-config.js
-USER nginx
+RUN chmod 666 /usr/share/nginx/html/app-config.js && \
+  chmod 664 /etc/nginx/* && \
+  chmod 775 /etc/nginx/conf.d && \
+  chown -R gen3:root /etc/nginx && \
+  yum install -y gettext && \
+  yum clean all
+
+USER gen3
```

Comment:
do we need to specify "USER gen3" twice?

Reply:
I'm changing the user from `gen3` to `root` in the above step to perform some permission management on folders the `gen3` user has no access to. Changing the permissions to `gen3` later.

Reply:
oh i missed that! thanks!